cmd/main.go

```diff
@@ -46,19 +50,24 @@ func init() {
 	//+kubebuilder:scaffold:scheme
 }
 
+const defaultExpirationTimeInHours = 7 * 12
```

Comment:
Why 7 * 12?

Reply:
This is just a random value that came up to my mind once I was doing quick experiments :).